game/src/engine/audio/AudioService.cpp

```diff
@@ -96,12 +98,14 @@ void AudioService::AddSource(std::string file_name, std::uint32_t entity_id)
     alSourcef(source, AL_MAX_DISTANCE, 300.0f);  // distance until silent
     alSourcef(source, AL_ROLLOFF_FACTOR, 0.5f);
 
+    SourceBufferPair source_buffer_pair = {source, buffer};
+
     if (alGetError() != AL_NO_ERROR)
     {
         Log::error("Couldn't create source for {}", file_name);
```

Comment:
Should we also assert or return or something here if we fail to create an audio source?

Reply:
i was wondering this too, my thinking was that failing to create an audio source isn't *that* serious of an error (?), it likely means that it couldn't find/open the file, and it *probably* (?) isn't worth to assert and crash just for that? i'll def add a return though, probably should've done that earlier!

Reply:
Yep, makes sense!